### PR TITLE
Document tzdata dependency

### LINKED ISSUE #4433: Document `tzdata` as a dependency
Certain features of CockroachDB require the `tzdata` package (sometimes called `zoneinfo`) to be installed (specifically, the ability to use place names as time zone identifiers). We had thought this was installed by default in all major linux distributions but in https://github.com/cockroachdb/cockroach/issues/32415#issuecomment-467196857 an ubuntu user reported that it was missing. We should document this dependency (and recommend that it be kept up to date, since the time zone/DST rules change several times a year).

### PR DESCRIPTION
**Pull request by jseldess**
Fixes #4433.

### CODE REVIEW

**Comment by jseldess** [comment]
@bdarnell, is this sufficient, or do we need to explicitly call this out in the install pages and/or the deployment tutorials?

File: v19.1/recommended-production-settings.md [hunk 1]
```diff
@@ -240,6 +240,10 @@ $ cockroach start --cache=.25 --max-sql-memory=.25 <other start flags>
 Avoid setting `--cache` and `--max-sql-memory` to a combined value of more than 75% of a machine's total RAM. Doing so increases the risk of memory-related failures.
 {{site.data.alerts.end}}
 
+## Time zone data
+
+Certain features of CockroachDB require the [`tzdata`](https://www.iana.org/time-zones) package (sometimes called `tz` or `zoneinfo`), for example, to support using place names as time zone identifiers. Therefore, ensure that this package is installed and kept up-to-date on all VMs where CockroachDB nodes are running.
```
Comment (bdarnell) [inline v19.1/recommended-production-settings.md]: "machines", not VMs - we don't know or care whether they're virtualized. 

This does not apply to docker-based deployments. The tzdata package is included in the docker image and is update with each patch release of cockroachdb.

Comment (jseldess) [reply v19.1/recommended-production-settings.md]: Done. I'll work on one more commit adding the `tzdata` error to the common errors page.

**Comment by jseldess** [comment]
@bdarnell, PTAL again.

I'm not sure why the build is failing. The anchors it's complaining about are perfectly fine.

**Pull request merged**